frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs

```diff
@@ -142,45 +145,89 @@
                      preview-complete?))
         (swap! state* assoc :selected-blend-mode current-blend-mode)))
 
-    [:div.element-set
-     [:div.element-set-title
-      [:span
-       (case type
-         :multiple (tr "workspace.options.layer-options.title.multiple")
-         :group (tr "workspace.options.layer-options.title.group")
-         (tr "workspace.options.layer-options.title"))]]
-
-     [:div.element-set-content
-      [:div.row-flex
-       [:& select
-        {:class "flex-grow no-check"
-         :default-value selected-blend-mode
-         :options options
-         :on-change handle-change-blend-mode
-         :is-open? option-highlighted?
-         :on-pointer-enter-option handle-blend-mode-enter
-         :on-pointer-leave-option handle-blend-mode-leave}]
-
-       [:div.input-element {:title (tr "workspace.options.opacity")
-                            :class "percentail"}
-        [:> numeric-input*
-         {:value (opacity->string current-opacity)
-          :placeholder (tr "settings.multiple")
-          :on-change handle-opacity-change
-          :min 0
-          :max 100}]]
-
-       [:div.element-set-actions.layer-actions
-        (cond
-          (or (= :multiple (:hidden values)) (not (:hidden values)))
-          [:div.element-set-actions-button {:on-click handle-set-hidden} i/eye]
-
-          :else
-          [:div.element-set-actions-button {:on-click handle-set-visible} i/eye-closed])
-
-        (cond
-          (or (= :multiple (:blocked values)) (not (:blocked values)))
-          [:div.element-set-actions-button {:on-click handle-set-blocked} i/unlock]
-
-          :else
-          [:div.element-set-actions-button {:on-click handle-set-unblocked} i/lock])]]]]))
+    (if new-css-system
+      [:div {:class (stl/css :element-set)}
+       [:div {:class (stl/css-case :element-set-content true
+                                   :hidden (or (:hidden values)
+                                               (:blocked values)))}
+        [:div {:class (stl/css :select)}
+         [:& select
+          {:default-value selected-blend-mode
+           :options options
+           :on-change handle-change-blend-mode
+           :is-open? option-highlighted?
+           :on-pointer-enter-option handle-blend-mode-enter
+           :on-pointer-leave-option handle-blend-mode-leave}]]
+        [:div {:class (stl/css :input)
+               :title (tr "workspace.options.opacity")}
+         [:span {:class (stl/css :icon)} "%"]
+         [:> numeric-input*
+          {:value (opacity->string current-opacity)
+           :placeholder (tr "settings.multiple")
+           :on-change handle-opacity-change
+           :min 0
+           :max 100
+           :className (stl/css :numeric-input)}]]
+
+
+        [:div {:class (stl/css :actions)}
+         (cond
+           (or (= :multiple (:hidden values)) (not (:hidden values)))
```

Comment:
If you use more than two times the same attribute lookup, prefer extracting it to a let binding.
Applies to  `(:hidden values)` and also to `(:blocked values)`